pkg/kyverno/test/test_command.go

```diff
@@ -148,6 +148,8 @@ skip  --> The rule is not applied.
 For more information visit https://kyverno.io/docs/kyverno-cli/#test
 `
 
+const KyvernoTestFile string = "kyverno-test.yaml"
```

Comment:
This is already defined in the command line defaults and it is the value that `fileName` takes. We shouldn't directly check for this specific fileName.

Reply:
Thanks